IHP/Controller/Redirect.hs

```diff
@@ -44,6 +47,18 @@ redirectToPath :: (?context :: ControllerContext) => Text -> IO ()
 redirectToPath path = redirectToUrl (fromConfig baseUrl <> path)
 {-# INLINABLE redirectToPath #-}
 
+-- | like 'redirectToPath', but forcing full page reload
+--
+-- Forces reload by using a custom HTTP OK header mimicking a HTTP redirect
+-- which is used as a signal to the AJAX call to perform page reload.
+-- currently this is a workaround of last resort when you can't make your Javscript 
+-- code behave properly together with morphdom and/or turbolinks
+-- 
+-- use 'forceRedirectToPath (pathTo action)' if you want to redirect to a controller action
+forceRedirectToPath :: (?context :: ControllerContext) => Text -> IO ()
+forceRedirectToPath path = respondAndExit $ Wai.responseLBS (Status 280 "IHP ForceRedirect") [(hLocation,TE.encodeUtf8 (fromConfig baseUrl <> path))] ""
```

Comment:
The more IHP-ish way would be to use `cs` instead of manually calling `encodeUtf8`. This also saves one import :)

```suggestion
forceRedirectToPath path = respondAndExit $ Wai.responseLBS (Status 280 "IHP ForceRedirect") [(hLocation,  cs (fromConfig baseUrl <> path))] ""
```